src/posix.mak

```diff
@@ -494,8 +497,11 @@ install: all
 
 ######################################################
 
-checkwhitespace: $(HOST_DMD_PATH)
-	CC=$(HOST_CXX) $(HOST_DMD_RUN) -run checkwhitespace.d $(SRC) $(GLUE_SRC) $(ROOT_SRCS)
+checkwhitespace: $(HOST_DMD_PATH) $(TOOLS_DIR)
```

Comment:
I thought that it didn't matter since the -run switch seems to delete the binary anyway after it runs it